### PR TITLE
Change callable

### PR DESCRIPTION
**Pull request by manjunath24**

### CODE REVIEW

**Comment by manjunath24** [comment]
@shabda : Added test cases for customer error message.

File: secure_login/tests.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 from .models import FailedLogin
 from .forms import SecureLoginForm, SecureFormMixin
+import secure_login.checkers as checkers
```
Comment (shabda) [inline secure_login/tests.py:12]: We should use a relative import here.

Comment (manjunath24) [inline secure_login/tests.py:12]: You mean like `from .checkers import no_weak_password` ? I noticed travis build failing for latest python version.

Comment (shabda) [inline secure_login/tests.py:12]: Yes.

**Pull request merged**